test/jdk/javax/swing/JTabbedPane/TestJTabbedPaneOpaqueColor.java

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.GridLayout;
+import javax.swing.AbstractAction;
+import javax.swing.AbstractButton;
+import javax.swing.JButton;
+import javax.swing.JCheckBox;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTabbedPane;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+import javax.swing.UnsupportedLookAndFeelException;
+
+/*
+ * @test
+ * @bug 8226990
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @summary Test JTabbedPane's contentarea, tab area and tab color
+ *          for different LAFs when opacity is enabled or disabled.
+ * @run main/manual TestJTabbedPaneOpaqueColor
+ */
+
+public class TestJTabbedPaneOpaqueColor {
+    private static JFrame frame;
+    private static JTabbedPane tabPane;
+    private static final String INSTRUCTIONS = """
+            The background color of panel (which contains the tabbed pane is green).
+            The background color of the tabbed pane is red.
+            The TabbedPane is not opaque initially.
+            For 'Content Opaque' and 'Tabs Opaque' to have effect, tab pane opacity should
+            be set to false i.e. Opaque checkbox should be unchecked.
+
+            Check the default behaviour of the tabbed pane:
+              - the area behind tabs is transparent (it must be green).
+              - the tabs area is opaque (it must be red, except the selected tab which must be gray).
```

Comment:
> As per my testing in windows, this is not satisfied for Nimbus and all tabs are gray, not only the selected one..
> Is this a bug not solved yet?

Don't think this as a bug. Since in Nimbus the tab colors and gradients are defined statically. So, this won't satisfy the Nimbus LAF.

> If it cannot be satisfied for Nimbus, it needs to be specified here upfront...
Ok.